pkg/config/config.go

```diff
@@ -43,30 +43,23 @@ const (
 
 	// FilePermissions is the default file permissions for the config file.
 	FilePermissions = 0o600
-
-	// RemoteEndpoint represents the API endpoint where we'll pull the dynamic
-	// configuration file from.
-	//
-	// NOTE: once the configuration is stored locally, it will allow for
-	// overriding this default endpoint.
-	RemoteEndpoint = "https://developer.fastly.com/api/internal/cli-config"
-
-	// UpdateSuccessful represents the message shown to a user when their
-	// application configuration has been updated successfully.
-	UpdateSuccessful = "Successfully updated platform compatibility and versioning information."
 )
 
-// ErrLegacyConfig indicates that the local configuration file is using the
-// legacy format.
-var ErrLegacyConfig = errors.New("the configuration file is in the legacy format")
+var (
+	CurrentConfigVersion int
```

Comment:
Let's get a doc string comment on this now it's public.

Reply:
Added.